pandas/core/strings.py

```diff
@@ -1316,6 +1358,27 @@ def str_encode(arr, encoding, errors="strict"):
     return _na_map(f, arr)
 
 
+def str_normalize(arr, form):
```

Comment:
Would getting rid of the not-strictly-relevant edits here in strings.py grease the wheels on this?